### ISSUE TITLE
plugins pack installed, D.B. crashes immediately

### ISSUE THREAD

**Issue by Marijn90** [Alexey-Yakovenko/deadbeef#1454]
Tested on CM7.2, CM10 and Miui on Motorola Atrix 4G.
D.B. works without plugin from playstore but as soon as it is installed it crashes.

Fixed permissions, wiped cache, different file systems etc to no avail...

Thx...

Calog:

```
01-22 00:19:41.005 I/ActivityManager(1680): Start proc org.deadbeef.android for service org.deadbeef.android/.MediaPlaybackService: pid=22788 uid=10010 gids={1015, 3003, 1028}
01-22 00:19:41.505 D/dalvikvm(22788): Trying to load lib /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so 0x40ea5790
01-22 00:19:41.505 D/dalvikvm(22788): Added shared lib /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so 0x40ea5790
01-22 00:19:41.505 I/DDB     (22788): dbconfdir: /storage/sdcard0/.deadbeef
01-22 00:19:41.505 I/DDB     (22788): loading plugins from /data/data/org.deadbeef.android/lib
01-22 00:19:41.505 I/DDB     (22788): load_plugin_dir /data/data/org.deadbeef.android/lib: scandir found 12 files
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libddb_dumb.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libdeadbeef.so
**01-22 00:19:41.505 I/DDB     (22788): dlsym error: Symbol not found:  (deadbeef_load)**
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libflac.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libm3u.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libmusepack.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libtta.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libvfs_curl.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libvfs_zip.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libvorbis.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android/lib/libwavpack.so
01-22 00:19:41.505 I/DDB     (22788): loading plugins from /data/data/org.deadbeef.android.freeplugins/lib
01-22 00:19:41.505 I/DDB     (22788): load_plugin_dir /data/data/org.deadbeef.android.freeplugins/lib: scandir found 14 files
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libaac.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libadplug.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libalac.so
01-22 00:19:41.505 I/DDB     (22788): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408): pid: 22788, tid: 22788, name: UNKNOWN  >>> org.deadbeef.android <<<
01-22 00:19:41.505 I/DEBUG   (1408):     #00  pc 00005718  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so (ffap_load+72)
01-22 00:19:41.505 I/DEBUG   (1408):     #01  pc 0002a45c  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (plug_init_plugin+24)
01-22 00:19:41.505 I/DEBUG   (1408):     #02  pc 0002b70c  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (plug_load_all+2448)
01-22 00:19:41.505 I/DEBUG   (1408):     #03  pc 00014ae8  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (Java_org_deadbeef_android_DeadbeefAPI_start+688)
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc34  5cf130b8  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc38  5cf0eb15  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc48  5cf130b8  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc50  5cf130b8  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc58  5cf130a0  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc64  5cf0f0db  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so (android_getCpuFamily+18)
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc7c  5cc04460  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (plug_init_plugin+28)
01-22 00:19:41.505 I/DEBUG   (1408):     #01  bee6fc80  5cc05700  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (plug_load_all+2436)
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fc88  5cf0e6d0  /data/data/org.deadbeef.android.freeplugins/lib/libffap.so (ffap_load)
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fcac  5cc05710  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so (plug_load_all+2452)
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fcbc  5cc1cdb8  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fcc8  5cc1cdc8  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fcd0  5cc1cdbc  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so
01-22 00:19:41.505 I/DEBUG   (1408):          bee6fcd8  5cc22e60  /mnt/asec/org.deadbeef.android-1/lib/libdeadbeef.so
01-22 00:19:41.505 I/DEBUG   (1408):     5cf08000-5cf09000 /data/data/org.deadbeef.android.freeplugins/lib/libalac.so
01-22 00:19:41.505 I/DEBUG   (1408):     5cf09000-5cf12000 /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:41.505 I/DEBUG   (1408):     5cf12000-5cf13000 /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
01-22 00:19:42.005 I/ActivityManager(1680): Process org.deadbeef.android (pid 22788) has died.
01-22 00:19:42.005 W/ActivityManager(1680): Canceling start item Intent { cmp=org.deadbeef.android/.MediaPlaybackService } in service ComponentInfo{org.deadbeef.android/org.deadbeef.android.MediaPlaybackService}
01-22 00:19:42.005 W/ActivityManager(1680): Scheduling restart of crashed service org.deadbeef.android/.MediaPlaybackService in 80000ms
_____________________________________________________________________________

01-22 00:43:29.505 D/OpenGLRenderer(1960): GL error from OpenGLRenderer: 0x505
01-22 00:43:31.005 I/DDB     (5883): dlsym error: Symbol not found:  (deadbeef_load)
01-22 00:43:31.505 W/InputDispatcher(1680): channel '4100a680 org.deadbeef.android/org.deadbeef.android.Deadbeef (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
01-22 00:43:36.505 I/DDB     (6065): dlsym error: Symbol not found:  (deadbeef_load)

_____________________________________________________________________________

01-22 00:42:29.505 F/libc    (4238): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4253 (AsyncTask #1)
01-22 00:42:30.005 F/libc    (4254): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4269 (AsyncTask #1)
01-22 00:42:30.005 F/libc    (4270): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4285 (AsyncTask #1)
01-22 00:42:31.005 F/libc    (4308): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4332 (AsyncTask #1)
**01-22 00:42:31.505 F/libc    (4295): Fatal signal 4 (SIGILL) at 0x5e0f3718 (code=1), thread 4295 (eadbeef.android)**
01-22 00:42:32.005 F/libc    (4341): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4356 (AsyncTask #1)
01-22 00:42:36.505 F/libc    (4401): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4417 (AsyncTask #1)
01-22 00:42:37.005 F/libc    (4419): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4436 (AsyncTask #1)
**01-22 00:42:37.505 F/libc    (4437): Fatal signal 4 (SIGILL) at 0x5cf0e718 (code=1), thread 4437 (eadbeef.android)**
01-22 00:42:38.505 F/libc    (4474): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4496 (AsyncTask #1)
01-22 00:42:39.505 F/libc    (4503): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4518 (AsyncTask #1)
01-22 00:42:40.005 F/libc    (4523): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4538 (AsyncTask #1)
01-22 00:42:41.005 F/libc    (4541): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4558 (AsyncTask #1)
01-22 00:42:41.505 F/libc    (4561): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4576 (AsyncTask #1)
01-22 00:42:42.005 F/libc    (4583): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4598 (AsyncTask #1)
01-22 00:42:42.505 F/libc    (4601): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4618 (AsyncTask #1)
01-22 00:42:45.005 F/libc    (4623): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4638 (AsyncTask #1)
01-22 00:42:45.505 F/libc    (4639): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4654 (AsyncTask #1)
01-22 00:42:46.005 F/libc    (4658): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4673 (AsyncTask #1)
01-22 00:42:47.005 F/libc    (4679): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4694 (AsyncTask #1)
01-22 00:42:47.505 F/libc    (4698): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4714 (AsyncTask #1)
01-22 00:42:48.005 F/libc    (4724): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4744 (AsyncTask #1)
01-22 00:42:49.005 F/libc    (4746): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4763 (AsyncTask #1)
01-22 00:42:49.505 F/libc    (4764): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4781 (AsyncTask #1)
01-22 00:42:50.005 F/libc    (4786): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4804 (AsyncTask #1)
01-22 00:42:50.505 F/libc    (4806): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4833 (AsyncTask #1)
01-22 00:42:51.005 F/libc    (4836): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4851 (AsyncTask #1)
01-22 00:42:52.005 F/libc    (4853): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4890 (AsyncTask #1)
01-22 00:42:52.505 F/libc    (4891): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4914 (AsyncTask #1)
01-22 00:42:53.005 F/libc    (4917): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4934 (AsyncTask #1)
01-22 00:42:55.505 F/libc    (4943): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4960 (AsyncTask #1)
01-22 00:42:56.005 F/libc    (4965): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 4981 (AsyncTask #1)
01-22 00:42:57.005 F/libc    (4983): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5000 (AsyncTask #1)
01-22 00:42:57.505 F/libc    (5004): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5019 (AsyncTask #1)
01-22 00:42:58.005 F/libc    (5020): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5035 (AsyncTask #1)
01-22 00:42:59.005 F/libc    (5040): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5055 (AsyncTask #1)
**01-22 00:42:59.005 F/libc    (5056): Fatal signal 4 (SIGILL) at 0x5cf0e718 (code=1), thread 5056 (eadbeef.android)**
01-22 00:43:00.505 F/libc    (5074): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5089 (AsyncTask #1)
01-22 00:43:01.505 F/libc    (5092): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5107 (AsyncTask #1)
01-22 00:43:02.005 F/libc    (5108): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5125 (AsyncTask #1)
01-22 00:43:02.505 F/libc    (5126): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5141 (AsyncTask #1)
01-22 00:43:03.005 F/libc    (5144): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5159 (AsyncTask #1)
01-22 00:43:05.505 F/libc    (5168): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5183 (AsyncTask #1)
01-22 00:43:06.505 F/libc    (5185): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5200 (AsyncTask #1)
01-22 00:43:07.005 F/libc    (5201): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5216 (AsyncTask #1)
01-22 00:43:07.505 F/libc    (5219): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5236 (AsyncTask #1)
01-22 00:43:08.005 F/libc    (5238): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5253 (AsyncTask #1)
01-22 00:43:09.005 F/libc    (5255): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5270 (AsyncTask #1)
01-22 00:43:09.505 F/libc    (5273): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5288 (AsyncTask #1)
01-22 00:43:10.505 F/libc    (5289): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5304 (AsyncTask #1)
01-22 00:43:11.005 F/libc    (5306): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5321 (AsyncTask #1)
01-22 00:43:11.505 F/libc    (5324): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5339 (AsyncTask #1)
01-22 00:43:12.005 F/libc    (5341): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5358 (AsyncTask #1)
01-22 00:43:12.505 F/libc    (5362): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5377 (AsyncTask #1)
01-22 00:43:14.505 F/libc    (5385): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5402 (AsyncTask #1)
01-22 00:43:15.505 F/libc    (5405): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5420 (AsyncTask #1)
01-22 00:43:16.005 F/libc    (5424): Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 5439 (AsyncTask #1)
```

**Comment by Alexey-Yakovenko** [comment]
Seems like android doesn't like that APE plugin is trying to detect CPU using the android_getCpuFamily call... I never heard this happening before for anyone else.

Unfortunately I can't look into this before march, sorry.

**Comment by Alexey-Yakovenko** [comment]
A simple fix would be to run the CPU detection code on demand, then deadbeef would only crash if APE file is played.

**Comment by Alexey-Yakovenko** [comment]
Googling on the topic reveals that this crash is very common when apps are built with NDK r10 of certain versions (a and b), so maybe updating NDK on my side would help.

**Comment by Marijn90** [comment]
Thanks for looking in to this. I'll keep trying once in a while.
Your help is being appreciated ;)

**Comment by Alexey-Yakovenko** [comment]
I uploaded the test build, which was made using NDK r11b, and possibly won't have the same issue anymore.
Could you please try it out?

https://sourceforge.net/projects/deadbeef/files/android/deadbeef.freeplugins-1.14-dev.apk/download

**Comment by Alexey-Yakovenko** [comment]
@Marijn90 hey, have you tried the fix?

**Comment by Marijn90** [comment]
Heyy, yes i tried but to no avail. Did some reboots and wiping stuff but it didnt get me any further.
It seems that it crashes now without plugin pack installed too :/

Any use if i post another catlog without the plugins installed on a new os?

Thanks,

Marijn

**Comment by Alexey-Yakovenko** [comment]
yes please.

**Comment by Alexey-Yakovenko** [comment]
@Marijn90 any news about this?

**Comment by Marijn90** [comment]
Well it works now but I've got different problems now related to the o.s. and not to deadbeef. Have both installed btw (updated plugins) on cm7.2.

I will be testing some more tomorrow and I hope it stays stable because it was the first thing I installed when going to a new o.s.

Catlog will be on its way too.

Thx & good night

**Comment by Marijn90** [comment]
04-07 22:00:08.591 I/run-parts(1313): Already checked: /data/app/org.deadbeef.android-1.apk
04-07 22:00:08.621 I/run-parts(1313): Already checked: /data/app/org.deadbeef.android.freeplugins-1.apk
04-07 22:06:28.548 I/ActivityManager(1560): Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=org.deadbeef.android/.Deadbeef } from pid 1715
04-07 22:06:28.568 I/ActivityManager(1560): Start proc org.deadbeef.android for activity org.deadbeef.android/.Deadbeef: pid=4269 uid=10030 gids={1015, 3003}
04-07 22:06:28.968 I/Database(4269): sqlite returned: error code = 0, msg = Recovered 14 frames from WAL file /data/data/org.deadbeef.android/databases/webview.db-wal
04-07 22:06:28.998 I/Database(4269): sqlite returned: error code = 0, msg = Recovered 6 frames from WAL file /data/data/org.deadbeef.android/databases/webviewCache.db-wal
04-07 22:06:29.078 D/dalvikvm(4269): Trying to load lib /data/data/org.deadbeef.android/lib/libdeadbeef.so 0x405140b0
04-07 22:06:29.098 D/dalvikvm(4269): DEX prep '/data/data/org.deadbeef.android/cache/ads1843884838.jar': unzip in 0ms, rewrite 164ms
04-07 22:06:29.118 D/dalvikvm(4269): Added shared lib /data/data/org.deadbeef.android/lib/libdeadbeef.so 0x405140b0
04-07 22:06:29.118 I/DDB     (4269): dbconfdir: /mnt/sdcard/.deadbeef
04-07 22:06:29.118 I/DDB     (4269): loading plugins from /data/data/org.deadbeef.android/lib
04-07 22:06:29.118 I/DDB     (4269): load_plugin_dir /data/data/org.deadbeef.android/lib: scandir found 12 files
04-07 22:06:29.118 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libddb_dumb.so
04-07 22:06:29.148 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libdeadbeef.so
04-07 22:06:29.148 I/DDB     (4269): dlsym error: Symbol not found:  (deadbeef_load)
04-07 22:06:29.148 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libflac.so
04-07 22:06:29.158 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libm3u.so
04-07 22:06:29.158 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libmusepack.so
04-07 22:06:29.168 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libtta.so
04-07 22:06:29.178 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libvfs_curl.so
04-07 22:06:29.188 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libvfs_zip.so
04-07 22:06:29.188 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libvorbis.so
04-07 22:06:29.198 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android/lib/libwavpack.so
04-07 22:06:29.208 I/DDB     (4269): loading plugins from /data/data/org.deadbeef.android.freeplugins/lib
04-07 22:06:29.208 I/DDB     (4269): load_plugin_dir /data/data/org.deadbeef.android.freeplugins/lib: scandir found 14 files
04-07 22:06:29.208 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libaac.so
04-07 22:06:29.228 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libadplug.so
04-07 22:06:29.238 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libalac.so
04-07 22:06:29.248 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libffap.so
04-07 22:06:29.248 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libgme.so
04-07 22:06:29.268 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libin_sc68.so
04-07 22:06:29.308 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libmms.so
04-07 22:06:29.318 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libmp3.so
04-07 22:06:29.318 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libsid.so
04-07 22:06:29.338 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libsndfile.so
04-07 22:06:29.358 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libvtx.so
04-07 22:06:29.368 I/DDB     (4269): loading plugin /data/data/org.deadbeef.android.freeplugins/lib/libwma.so
04-07 22:06:29.538 I/DDB     (4269): DeadbeefAPI.start returned 0
04-07 22:06:29.538 I/AudioService(1560):   Remote Control   registerMediaButtonEventReceiver() for ComponentInfo{org.deadbeef.android/org.deadbeef.android.MediaButtonIntentReceiver}
04-07 22:06:29.588 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.588 I/AudioService(1560):  AudioFocus  abandonAudioFocus() from android.media.AudioManager@405b1698org.deadbeef.android.Player$AudioFocusHelper@405ba6f8
04-07 22:06:29.788 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.798 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.798 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.798 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.798 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:29.848 I/ActivityManager(1560): Displayed org.deadbeef.android/.Deadbeef: +1s289ms
04-07 22:06:31.618 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.628 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.628 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.628 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.638 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.638 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.648 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.668 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:31.678 I/AudioService(1560):  AudioFocus  requestAudioFocus() from android.media.AudioManager@405b1698org.deadbeef.android.Player$AudioFocusHelper@405ba6f8
04-07 22:06:34.048 I/ActivityManager(1560): Starting: Intent { cmp=org.deadbeef.android/.EQ } from pid 4269
04-07 22:06:34.358 I/ActivityManager(1560): Displayed org.deadbeef.android/.EQ: +211ms
04-07 22:06:35.238 I/ActivityManager(1560): Starting: Intent { cmp=org.deadbeef.android/.SelectEqPreset } from pid 4269
04-07 22:06:35.438 I/ActivityManager(1560): Displayed org.deadbeef.android/.SelectEqPreset: +126ms
04-07 22:06:39.168 I/ActivityManager(1560): Starting: Intent { cmp=org.deadbeef.android/.SelectEqPreset } from pid 4269
04-07 22:06:39.398 I/ActivityManager(1560): Displayed org.deadbeef.android/.SelectEqPreset: +148ms
04-07 22:06:52.498 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.498 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.498 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.498 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.508 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.508 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.508 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:52.508 I/DDB     (4269): Sending com.deadbeef.widget.INFO
04-07 22:06:58.798 I/ActivityManager(1560): Starting: Intent { act=org.deadbeef.android.PLAYBACK_VIEWER flg=0x10000000 cmp=org.deadbeef.android/.Deadbeef bnds=[0,136][540,232] } from pid -1
04-07 22:07:03.468 I/ActivityManager(1560): Starting: Intent { cmp=org.deadbeef.android/.Settings } from pid 4269
04-07 22:07:03.658 I/ActivityManager(1560): Displayed org.deadbeef.android/.Settings: +154ms

**Comment by Marijn90** [comment]
I cant read them but i hope this makes clear if the problem is solved. As it seems now the player is stable for me and ive got no problems. I am running android without gapps now it that makes a difference.

Ill close the issue as soon as i hear from you ;)

Last question... do the older android versions stay supported or are you gonna ad functions that require a new android version?

Btw... the progress bar of the song playing... was it always  like this? I thought i was smoother the last time i was able to use it.

big THX

Marijn

**Comment by Alexey-Yakovenko** [comment]
@Marijn90 some of your questions imply that I changed something. The only thing I really did was that I rebuilt the plugins with the latest version of Android NDK.

I can't see from the log if any problems are solved or not, but there's no crash information, so I guess it started working.

> do the older android versions stay supported or are you gonna ad functions that require a new android version?

The way it works in Google play store, is that even if I stop, or Google prevents to, shipping releases for older Android versions -- it would still be possible to get old versions of the App, compatible with your device. This means, you won't get updates anymore, but the app will work.

Regarding the future versions of Deadbeef -- supporting Android 2.3 depends on demand, and whether Google supports it, etc. Currently, Android 2.3 is the oldest version supported by google libraries. If Google decides it should only support Android 4.0+ -- there wouldn't be much I can do.
